File: C++/freeCodeCamp.org_C++ Tutorial for Beginners/src/016_Switch Statements/SwitchStatements.cpp
#include <iostream>

/* Simple C++ program that uses switch statements */

using namespace std;

string getDayOfWeek(int);

int main()
{
    cout << getDayOfWeek(0) << endl;

    return 0;
}

string getDayOfWeek(int dayNum)
{
    string dayName;
    
    switch(dayNum)
    {
        case 0: // If dayNum is 0 execute this code
            dayName = "Sunday";
            break;
        case 1:
            dayName = "Monday";
            break;
        case 2:
            dayName = "Tuesday";
            break;
        case 3:
            dayName = "Wednesday";
            break;
        case 4:
            dayName = "Thursday";
            break;
        case 5:
            dayName = "Friday";
            break;
        case 6:
            dayName = "Saturday";
            break;
        default: // If it is none of the cases execute this code
            dayName = "[ERROR]: Invalid day number";
            break;
    }

    return dayName;
}

File: Python/002_OOP Basics/016_Switch Statements/SwitchStatements.py
def getDayOfWeek(dayNum):
    if(dayNum == 0): # If dayNum is 0 execute this code
        return "Sunday"
    elif(dayNum == 1):
        return "Monday"
    elif(dayNum == 2):
        return "Tuesday"
    elif(dayNum == 3):
        return "Wednesday"
    elif(dayNum == 4):
        return "Thursday"
    elif(dayNum == 5):
        return "Friday"
    elif(dayNum == 6):
        return "Saturday"
    else: # If it is none of the cases execute this code
        return "[ERROR]: Invalid day number"

print(getDayOfWeek(0))